paddle/operators/gaussian_random_op.cu

```diff
@@ -33,7 +33,8 @@ class GaussianRandomKernel : public framework::OpKernel {
 
     int seed = context.op_.GetAttr<int>("seed");
     if (seed == 0) {
-      seed = std::random_device()();
+      std::random_device rd;
```

Comment:
The old way will cause 'type name' declared as function returning a function error